postgresql-s3-backup/README.md

```diff
@@ -0,0 +1,24 @@
+Creation of the Cronjob
+- the cronjob is create to make regular backups once at between 00 : 00 and 01 : 00.
+- it is created to make buckups of postgres database data.
+- integrated with amazon-s3 bucket
+- docker image that we used
+- https://github.com/schickling/dockerfiles/tree/master/postgres-backup-s3
+
+Valid parameters for postgres-s3-backup
+Parameter                 | Values
+---------------------------------------------------------------------------------------------------------------------------------------------------------------- 
+- schedule                | '@daily'  *** set the schedule for the cronjob to run at e.g ' 0 0 */1 * *' will run every day at 00 : 00
+- s3.region               | region *** which region is the S3 bucket set
+- s3.accessKeyId          | key *** the access key id to the S3 bucket that you have (NOTE should be provided from a secret, DO NOT PUSH THIS TO A PUBLIC REPO)
```

Comment:
ah I messed up. If you provide a secret (which you must with s3.secret).
Then it does not make sense to have these parameters